website/src/pages/docs/guides/hive.mdx

```diff
@@ -66,6 +66,19 @@ plugins:
 ```
 
 <Callout>You have to create a project on GraphQL Hive and get an access token.</Callout>
+<Callout>If you want to collect usage data for the [Insights tab](https://the-guild.dev/graphql/hive/product-updates/2023-09-28-introducing-graphql-api-consumer-insights-page) in hive, you must include the `usage` attribute in the hive section of the configuration.
+
+For example:
+
+```yaml filename=".meshrc.yaml"
+plugins:
+  - hive:
+      token: '{env.HIVE_TOKEN}'
+      usage: {}
```

Comment:
Actually `usage` is enabled by default currently in Hive plugin;
https://github.com/ardatan/graphql-mesh/blob/master/packages/plugins/hive/src/index.ts#L33

Reply:
Oh, woops - recent change. My bad!

I'll close this one